dashboard/src/pages/ChapterMembers.vue

```diff
@@ -20,7 +24,10 @@
         v-for="member in chapter.doc.chapter_members"
         :title="member.full_name"
       >
-        <template v-if="member.email != session.user" #actions>
+        <template
+          v-if="member.email != session.user && member.role != 'Lead'"
+          #actions
+        >
```

Comment:
![Screenshot 2024-08-10 at 2 12 34 AM](https://github.com/user-attachments/assets/e6c69410-4fe8-4b34-8a32-514876c420dc)

Any member other than Lead shouldn't have the option to remove another member. They however can have the permission to leave one's own.

Reply:
Question: what if the lead leaves? We don't want a deadlock. 

Also, there's no provision to handle role change. When we create something for role change, we can make this change. 

Won't be doing this in current PR

Reply:
The same case was mentioned in this issue - #334

Reply:
Not taking up in this PR. Will make changes with the feature of changing roles

Reply:
Approving the PR since the issue already exists on that note.